src/main/java/Berry.java

```diff
@@ -0,0 +1,57 @@
+import java.util.Scanner;
+
+public class Berry {
+    public static void main(String[] args) {
+        String chatbotName = "Berry";
+        String userCommand;
+        Task[] tasks = new Task[100];   //array of Task objects
+        int numberOfTasks = 0;
+        Scanner in = new Scanner(System.in);
+        System.out.println("‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧\nHello! I'm " + chatbotName
+                + "\nWhat can I do for you?\n‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧\n");
+        userCommand = in.nextLine();  //get command from user
+        while (!userCommand.equals("bye")) {
+            if (userCommand.equals("list")) {
+                printList(tasks, numberOfTasks);
+            } else if (userCommand.contains("mark")) {
+                toggleTaskStatus(userCommand, tasks);
+            } else {
+                addTask(userCommand, tasks, numberOfTasks);
+                numberOfTasks++;
+            }
+            userCommand = in.nextLine();
+        }
+        System.out.println("\n‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧\nBye. Hope to see you again soon!"
+                + "\n‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧");
+    }
+
+    public static void addTask(String command, Task[] tasks, int numberOftasks) {
+        tasks[numberOftasks] = new Task(command);
+        System.out.println("\n‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧\nadded: " + command + "\n‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧\n");
+    }
+
+    public static void printList(Task[] tasks, int numberOftasks) {
+        System.out.println("\n‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧‧⋆ ✧˚₊‧⋆. ✧˚₊‧⋆‧");
+        for (int i = 0; i < numberOftasks; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < numberOfTasks; i++) {
```
The variable numberOftasks doesn’t follow Java naming conventions. Variable names should be in camelCase (so it should be numberOfTasks), and in this case, taskCount or simply size would be clearer and more consistent with Java’s collection classes.